src/schema/artist/current.js

```diff
@@ -0,0 +1,87 @@
+import { GraphQLObjectType, GraphQLString } from "graphql"
+import Image from "schema/image"
+import { error } from "lib/loggers"
+import moment from "moment"
+import { exhibitionPeriod } from "lib/date"
+
+const CurrentEventType = new GraphQLObjectType({
+  name: "CurrentEvent",
+  fields: {
+    image: {
+      type: Image.type,
+    },
+    headline: {
+      type: GraphQLString,
+    },
+    subHeadline: {
+      type: GraphQLString,
+    },
+    name: {
+      type: GraphQLString,
+    },
+  },
+})
+
+const showSubHeadline = show => {
+  let headline = show.partner + "\n"
```

Comment:
Ok, this is silly, but the comp has two lines for a show, and one line for an auction (which are spans).

How can I get the React component to be able to render a new line conditionally for that? I thought adding it here would work, but  I'm not sure. An alternative would be to have a `+ "<br />"` kind of thing and use `dangerouslySetHtml`? I'll defer to you.

Reply:
Not silly -- clear :) `\n` jumps right out as  a new line